bot.py

```diff
@@ -63,6 +65,42 @@ def get_token():
 		return "".join(f.readlines()).strip()
 
 
+@bot.command()
+async def minigame(ctx: commands.Context):
+	"""A minigame to allow users to aquire Pokemon"""
+	await lock.acquire()
```

Comment:
If you do this, then you won't be able to run multiple minigames in multiple discord channels at the same time.

Reply:
This is what you suggested I do when we were talking about this issue the other day. What would you suggest as an alternative?

Reply:
I would suggest a lock per channel. It might make more sense to just not allow multiple minigames to happen in the same channel at the same time, because having a lock would mean that as soon as the current minigame ended, the next one would start. That could be weird from a UX perspective